lib/plugins/aws/package/lib/strip-null-props-from-template-resources.js

```diff
@@ -15,6 +15,9 @@ module.exports = {
     const resources = this.serverless.service.provider.compiledCloudFormationTemplate.Resources;
 
     for (const resource of Object.values(resources)) {
+      if (resource.Properties !== undefined && resource.Properties === null) {
```

Comment:
Let's simplify to `resource.Properties == null` (there's no valid reason to treat `null` and `undefined` differently)

Reply:
There can be different scenarios:

ParentObject: 
-> no Properties defined -> undefined

ParentObject:
  Properties:
-> Properties defined but = null

I am trying to catch the second one, and skip the 1st one
from the top of my head, `resource.Properties == null` will fall into both

Another way to write the condition would be:
```
if ('Properties' in resource && resource.Properties === null) {
```
or
```
if (resource.hasOwnProperty('Properties') && resource.Properties === null) {
```

Reply:
`resource.Properties == null` will catch all this scenarios, and that's wanted

Reply:
`resource.Properties == null` will catch all of those scenarios, and that's wanted

Reply:
hi @medikoo 

Added one test case for Properties not declared. Updated my code to follow accordingly. Can verify using
`npx mocha test/unit/lib/plugins/aws/package/lib/strip-null-props-from-template-resources.test.js`